docs/README.md

```diff
@@ -0,0 +1,57 @@
+# Managing Lacework Integrations and Configuration with Hashicorp Terraform
+For companies who have adopted Terraform for automation within their organization, Lacework maintains two open source projects to support managing Lacework organizations and accounts as code including the integrations with AWS, GCP, and Azure cloud environments. The two projects are:
```

Comment:
nitpick: just because GCP means Google Cloud Platform
> with AWS, **Google**, and Azure cloud environments

Reply:
@afiune Do you think we should standardize on Google or Google Cloud Platform?

Reply:
I think we could use GCP when we don't refer to the other cloud providers. Otherwise, I would say lets go with Google Cloud. Thoughts?

Reply:
sounds good.